dev/bots/run_command.dart

```diff
@@ -80,7 +80,7 @@ Future<void> runCommand(String executable, List<String> arguments, {
   );
 
   final String commandDescription = '${path.relative(executable, from: workingDirectory)} ${arguments.join(' ')}';
-  final String relativeWorkingDir = path.relative(workingDirectory);
+  final String relativeWorkingDir = path.relative(workingDirectory ?? Directory.current.path);
```

Comment:
This throws right now if someone calls it without specifying a working directory.  If that's intended, we can change this to be an `@required` param, but I think this makes sense.